### ISSUE TITLE
Unable to run via npm dependency but working if I copy files directly

### ISSUE THREAD

**Issue by etsrepo** [neilff/redux-ui-router#69]
I dont know what messing in my configuration of structure but If I install as npm dependency its not its not triggering when states changes.

but if I copy this projects files its working properly.

I'm tried with your example also bahavior is same.

I can't post my code so attaching images 

working one (redux-router is close of this repo)
![image](https://cloud.githubusercontent.com/assets/3172168/18664661/3b521ec4-7f41-11e6-9fcf-0ce38affd6be.png)

not working (see after from in import)
![image](https://cloud.githubusercontent.com/assets/3172168/18664680/5406dedc-7f41-11e6-905b-9c25a6362435.png)

same with your example directory 

working 
![image](https://cloud.githubusercontent.com/assets/3172168/18664780/ba09ef58-7f41-11e6-9f8e-c94e5464300d.png)

not working 
![image](https://cloud.githubusercontent.com/assets/3172168/18664792/cb313c6e-7f41-11e6-88a2-23de0cb2bac8.png)

log out for working when importing from cloned files
![image](https://cloud.githubusercontent.com/assets/3172168/18664891/36a969f8-7f42-11e6-8343-256e2895da6f.png)

not working here if importing as npm dependency
![image](https://cloud.githubusercontent.com/assets/3172168/18664940/6e2b2006-7f42-11e6-929c-b8ac57cfaeec.png)

can you please tell me, what I'm doing wrong?

**Comment by etsrepo** [comment]
I'm using 0.6.0 version from npm... will 0.6.1 solve this issue ?? anyhow its not published yet

**Comment by hally9k** [comment]
Hi @etsrepo, Which version of ui-router have you got installed?

**Comment by hally9k** [comment]
Unable to reproduce this end so I've closed this for now. Feel free to pick it back up at any time.

**Comment by etsrepo** [comment]
```
    "angular-ui-router": "^1.0.0-beta.2",
    "angular": "^1.5.8",
    "ng-redux": "^3.4.0-beta.1",
    "react-redux": "^4.4.5",
    "redux": "^3.6.0",
    "redux-devtools": "^3.3.1",
    "redux-logger": "^2.6.1",
    "redux-thunk": "^2.1.0",
    "redux-ui-router": "^0.6.0"
```

**Comment by hally9k** [comment]
@etsrepo - The version 0.6.0 doesn't have support for angular-ui-router 1.0 beta. The current state of master, which is marked 0.6.1, will be released as 0.7.0 because it brings support for angular-ui-router 1.0 but we won't release this until angular-ui-router 1.0 comes out of beta, I expect. If you roll back your angular-ui-router version to 0.3.1 then redux-ui-router 0.6.0 will work for you. Sorry about that. Maybe this should be more clear in the docs and an explicit peer dependency... Hmmm I'll have a think about it. Hope this helps!
